docs/system-metrics.md

```diff
@@ -0,0 +1,57 @@
+# System-metrics collection
+
+Benchmark-operator is able to collect prometheus metrics from the cluster at the end of a benchmark. To do so, it creates a k8s job that uses [kube-burner](https://github.com/cloud-bulldozer/kube-burner) to collect the Prometheus metrics given by a configuration file. This system metrics collection mechanism in available for all workloads except `kube-burner` and `backpack`.
+
+This feature is disabled by default, it can be enabled by adding a `system_metrics` section to a benchmark CR.
+
+```yaml
+  system_metrics:
+    collection: true (Defaults to false)
+    prom_url: <Valid prometheus endpoint, by default https://prometheus-k8s.openshift-monitoring.svc.cluster.local:9091>
+    index_name: <ES index name, defaults to system-metrics>
+    es_url: <valid ES endpoint>
+    prom_token: <valid prometheus token, in an OpenShift environment can be obtained with oc sa get-token -n openshift-monitoring prometheus-k8s>
+    metrics_profile: <valid metric profile name or URL pointing to it, by default node-metrics.yml>
+	step: <Prometheus step size, by default 30s>
```

Comment:
Something weird going on here with the spacing?

Reply:
Thanks, fixed